docs/sources/flow/reference/cli/run.md

```diff
@@ -134,6 +135,12 @@ The first node that is used to bootstrap a new cluster (also known as
 the "seed node") can either omit the flags that specify peers to join or can
 try to connect to itself.
 
+The `--cluster.name` flag can be used to prevent clusters from accidentally merging.
+When `--cluster.name` is provided, nodes will only join peers who share the same cluster name value.
+By default, the cluster name is empty, and any node that doesn't set the flag can join.
+Attempting to join a cluster with a wrong `--cluster.name` will result in a "failed to join memberlist" error
+with a `EOF` reason.
```

Comment:
```suggestion
Attempting to join a cluster with a `--cluster.name` that doesn't match will result in a "failed to join memberlist" error
with an `EOF` reason.
```

That reason isn't very intuitive. Is it possible to make it more explicit - something like "Cluster name not matched"?

What does the actual error message look like?

Reply:
The problem is that the check is done in memberlist (hashicorp repo) and that they don't return the error. The actual message is `ts=2023-08-29T08:08:59.457373Z level=warn msg="failed to connect to peers; bootstrapping a new cluster" err="failed to join memberlist: 2 errors occurred:\n\t* Failed to join [::1]:12345: EOF\n\t* Failed to join 127.0.0.1:12345: EOF\n\n"`.
So in the agent and in ckit we don't have the control over it, we just know that the node tried to join the memberlist and that something went wrong

Reply:
Ah ok. Then is it necessary to mention EOF?  Can we just note `failed to join memberlist`?  There's other info that we don't note like the specific nodes...

Reply:
you're right, it's probably better to just keep `failed to join memberlist`, I will update it